File: CS245/Lab3/FizzBuzz.go
package main
import "fmt"

func main() {
	count := 0
        for i := 0; i < 100; i++ {
            count++;
            if ((count%3) == 0) && ((count%5) == 0){
                fmt.Println("FizzBuzz");
            } else if (count%5) == 0 {
                fmt.Println("Buzz");
            } else if (count%3) == 0 {
                fmt.Println("Fizz");
            } else {
                fmt.Println(count);
            }
        }
}

File: CS245/Lab2/FizzBuzz.java
public class FizzBuzz {
    public static void main (String[] args) {
        int count = 0;
        for (int i = 0; i < 100; i++) {
            count++;
            if (((count%3) == 0) && ((count%5) == 0)){
                System.out.println(count + " FizzBuzz");
            }
            else if ((count%5) == 0) {
                System.out.println(count + " Buzz");
            }
            else if ((count%3) == 0) {
                System.out.println(count + " Fizz");
            }
            else {
                System.out.println(count);
            }
        }
    }
}